drools-wb-services/drools-wb-verifier/drools-wb-verifier-core/src/main/java/org/drools/workbench/services/verifier/core/checks/DetectRedundantActionCheck.java

```diff
@@ -73,7 +75,7 @@ public Issue getIssue() {
                                                          new HashSet<>( Arrays.asList( ruleInspector.getRowIndex() + 1 ) ) );
         } else {
             return new ValueForActionIsSetTwiceIssue( Severity.WARNING,
-                                                      ExplanationType.VALUE_FOR_ACTION_IS_SET_TWICE,
+                                                      CheckType.VALUE_FOR_ACTION_IS_SET_TWICE,
```

Comment:
Yes, I mean, there is used correct CheckType at the commented line, but probably it would be worth not to throw two kinds of issues from one kind of checker.